src/zope/annotation/attribute.py

```diff
@@ -30,16 +31,22 @@
 
 _EMPTY_STORAGE = _STORAGE()
 
+ATTR = "_zope_annotations"
```

Comment:
Can we have this as a class attribute?

That would make it easier to come up with a customized adapter that uses another value.